parts/header.html

```diff
@@ -1,9 +1,9 @@
 <!-- wp:group {"layout":{"inherit":true}} -->
 <div class="wp-block-group"><!-- wp:group {"align":"wide","layout":{"inherit":false}} -->
-<div class="wp-block-group alignwide"><!-- wp:loginout /--></div>
-<!-- /wp:group -->
+	<div class="wp-block-group alignwide"><!-- wp:loginout /--></div>
+	<!-- /wp:group -->
 
-<!-- wp:site-title {"align":"wide"} /-->
+	<!-- wp:site-title {"align":"wide"} /-->
 
-<!-- wp:navigation {"ref":12,"align":"wide","layout":{"type":"flex","setCascadingProperties":true,"flexWrap":"nowrap"},"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}}} /--></div>
-<!-- /wp:group -->
\ No newline at end of file
+	<!-- wp:navigation {"ref":12,"align":"wide","layout":{"type":"flex","setCascadingProperties":true,"flexWrap":"nowrap"},"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}}} /--></div>
```

Comment:
What is this meant to reference? Can this be removed?

Reply:
This is a placeholder for a navigation element. It was markup that actually was pulled from the existing gesso branch header.html. This can be removed if we want the template to just be blank and people can manage it from scratch, but I started a few other files with the boilerplate from the current gesso 5 branch too.